version/VERSION

```diff
@@ -1 +1 @@
-0.7.2
+0.7.3-rc
```

Comment:
This feels ever so slightly off. Is it possible to build it without updating this to add the -rc suffix?

Reply:
My thinking is so we know whether we should do stable (public) or private release.
If we don't want `-rc` suffix, we could check if the tag for that version exists or not, if tag already exists, then we to private release, otherwise it's a stable release. But it means that changes that will be released in `0.7.3` will have a version `0.7.2-123` instead. Thoughts?

Reply:
I think this is fine - this appears to be the way that the agent is doing it as well: https://github.com/buildkite/agent/blob/main/version/VERSION